sdk/communication/Azure.Communication.Email/api/Azure.Communication.Email.netstandard2.0.cs

```diff
@@ -1,171 +1,132 @@
 namespace Azure.Communication.Email
 {
-    public static partial class CommunicationEmailModelFactory
```

Comment:
Missing the [Asp net extensions](https://azure.github.io/azure-sdk/dotnet_implementation.html#integration-with-aspnet-core).

Reply:
added asp.net extesions as per the linked guidelines.